src/alto-ui/Dropdown/Dropdown.js

```diff
@@ -212,6 +212,7 @@ class Dropdown extends React.Component {
 
   render() {
     const {
+      id,
```

Comment:
add `id` to deconstruction because this `id` was passed down to children and all the children have the same `Id`